docs/en/reference/improving-performance.rst

```diff
@@ -26,6 +26,8 @@ Doctrine will need to load your mapping information on every single
 request and has to parse each DQL query on every single request.
 This is a waste of resources.
 
+See :ref:`integrating-with-the-orm`
```

Comment:
https://github.com/doctrine/orm/blob/b243ea1940d65d9ec3cc3045286726b7e79145b7/docs/en/reference/caching.rst#integrating-with-the-orm